linkerd/app/inbound/src/policy.rs

```diff
@@ -38,9 +41,19 @@ pub struct ServerUnauthorized {
     server: Arc<Meta>,
 }
 
+/// Returns the traffic policy configured for the destination address.
 pub trait GetPolicy {
-    // Returns the traffic policy configured for the destination address.
-    fn get_policy(&self, dst: OrigDstAddr) -> AllowPolicy;
+    type Error: Into<Error>;
```

Comment:
What are the tradeoffs between including an Error type attribute and hardcoding the `Error` type. It seems like it could be unnecessary boilerplate for non-library use cases.

Reply:
this was lifted from `GetProfile` for consistency. it's probably better off just always turning the error into an `Error` though.